media-libs/libshumate/libshumate-1.0.1.ebuild

```diff
@@ -0,0 +1,54 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gnome.org meson vala
+
+DESCRIPTION="Shumate is a GTK toolkit providing widgets for embedded maps"
+HOMEPAGE="https://wiki.gnome.org/Projects/libshumate"
+
+SLOT="0"
+LICENSE="LGPL-2.1+"
+KEYWORDS="~amd64"
+REQUIRED_USE="gtk-doc? ( introspection )"
+
+IUSE="introspection gtk-doc vala" # vector-renderer is still experimental, maybe put in at a later release
+
+RDEPEND="
+	>=dev-libs/glib-2.68.0:2
+	>=x11-libs/cairo-1.4
+	>=dev-db/sqlite-1.12:3
+	>=gui-libs/gtk-4:4
+	>=net-libs/libsoup3-3:3
```

Comment:
Please run `pkgcheck scan --commits`. It will catch mistakes like this. It's the same tool as is run in the CI that leaves comments on PRs.